### PR TITLE
Start rename to HospitalHero

### PR DESCRIPTION
**Pull request by cantino**
Not renaming the repo, org, or heroku / circle resources yet. Just text in this PR.

### CODE REVIEW

File: app/javascript/packs/data/socialMediaLinks.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 export const FACEBOOK_LINK =
-  "https://www.facebook.com/sharer/sharer.php?u=https%3A//feedadoc.com/";
+  "https://www.facebook.com/sharer/sharer.php?u=https%3A//http://hospitalhero.care/";
```
Comment (tuntisz) [inline app/javascript/packs/data/socialMediaLinks.js]: looks like these links are `https://http://hospitalhero.care` if I'm reading that correctly

Comment (cantino) [reply app/javascript/packs/data/socialMediaLinks.js]: Thanks!

File: app/views/layouts/mailer.html.erb [hunk 2]
```diff
@@ -475,7 +475,7 @@
                   <tr>
                     <td class="content-cell" align="center" style="word-break: break-word; font-family: &quot;Poppins&quot;, Helvetica, Arial, sans-serif; font-size: 16px; padding: 45px;">
                       <p class="f-fallback sub align-center" style="font-size: 13px; line-height: 1.625; text-align: center; color: #A8AAAF; margin: .4em 0 1.1875em;" align="center">
-                        <a href="https://feedadoc.com">Feed A Doc</a>
+                        <a href="http://hospitalhero.care/">HospitalHero</a>
```
Comment (tuntisz) [inline app/views/layouts/mailer.html.erb]: minor: `http` -> `https`

**Review by tuntisz — APPROVED** [review]
:hospital:

**Pull request merged**